components/Nav/Nav.tsx

```diff
@@ -13,7 +13,7 @@ const navigation = [
 
 const Nav: NextPage = () => {
   return (
-    <Popover as="header" className="relative">
+    <Popover as="header" className="relative" style={{zIndex: 10}}>
```

Comment:
use tailwindcss classes for z-index

Reply:
To second Praveen - [https://tailwindcss.com/docs/z-index](https://tailwindcss.com/docs/z-index)

z-10 should be enough